app/views/tickets/_ticket.html.haml

```diff
@@ -8,8 +8,12 @@
         - unless ticket.description.blank?
           = markdown(ticket.description)
   %td.col-sm-1.col-md-1
-    = text_field_tag("tickets[][#{ticket.id}]", 0,
-      type: 'number', min: 0, class: "form-control quantity", 'data-id' => ticket.id)
+    - if ticket.registration_ticket?
+      = text_field_tag("tickets[][#{ticket.id}]", 0,
+        type: 'number', min: 0, max: 1, class: "form-control quantity", 'data-id' => ticket.id)
```

Comment:
couldn't you introduce the if directly in the max or setting the params? Most of the code here is duplicated. :thinking: